.github/buildEverything.py

```diff
@@ -91,8 +91,13 @@ def doBuild(makeDebug):
 	
     ap.add_argument("-l", "--support-language", type=str, required=True, help="# model language to compile. (Enter 'all' to compile all language packs.)")
     ap.add_argument("-m", "--support-model", type=str, required=True, help="# model type to compile. (Enter 'all' to compile all model packs.)")
+    ap.add_argument("-f", "--support-feature", type=str, required=False, default="", help="# feature to compile. (Enter a feature to compile.)")
     args = vars(ap.parse_args())
 
+    if str(args["support_feature"]) != "":
+        logging.info('Setting flag ' + str(args["support_feature"]) + '...')
+        os.system('sed -i'' -r -e "/build_flags/a\    -D '+str(args["support_feature"])+'" platformio.ini')
```

Comment:
convert `Win cmd`?

Reply:
Why did you ever use `os.system`? I think even here we used code to open the `platformio.ini` and to read it!